--- cppzmq-4.2.2/libzmqPkgConfigFallback.cmake.orig	2017-07-31 15:30:55.000000000 +0200
+++ cppzmq-4.2.2/libzmqPkgConfigFallback.cmake	2017-12-25 14:33:02.688755906 +0100
@@ -15,6 +15,6 @@
 set_property(TARGET libzmq-static PROPERTY INTERFACE_INCLUDE_DIRECTORIES ${PC_LIBZMQ_INCLUDE_DIRS})
 set_property(TARGET libzmq-static PROPERTY IMPORTED_LOCATION ${ZeroMQ_STATIC_LIBRARY})
 
-if(ZeroMQ_LIBRARY AND ZeroMQ_STATIC_LIBRARY)
+if(ZeroMQ_LIBRARY)
     set(ZeroMQ_FOUND ON)
 endif()
